server/Cargo.lock

```diff
@@ -150,10 +150,10 @@ source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "f4ebf134c444f81a8f49f315792cede0fcfb489d8a434d14a60c4e06ae6e1624"
 dependencies = [
  "dirs",
- "reqwest",
+ "reqwest 0.10.6",
```

Comment:
maybe you should `cargo build` across:
+ [ ] cli
+ [ ] linux
+ [ ] server

so that the lock files update in this PR rather than someone else's pr